K2Bridge/Visitors/IVisitor.cs

```diff
@@ -133,6 +133,18 @@ internal interface IVisitor
         /// <param name="queryStringClause">The query string clause.</param>
         void Visit(QueryStringClause queryStringClause);
 
+        /// <summary>
+        /// Accepts a given visitable object and builds a Kusto query.
+        /// </summary>
+        /// <param name="rangeAggregation">The Range aggregation to visit.</param>
+        void Visit(DateRangeAggregation dateRangeAggregation);
+
+        /// <summary>
+        /// Accepts a range expression, and builds the Kusto query.
```

Comment:
/// Accepts a given visitable object and builds a Kusto query. <- To be aligned with other docstrings

Reply:
👍🏻